src/Symfony/Component/DomCrawler/Tests/CrawlerTest.php

```diff
@@ -56,13 +60,23 @@ public function testAdd()
     /**
      * @expectedException \InvalidArgumentException
      */
-    public function testAddInvalidNode()
+    public function testAddInvalidType()
     {
         $crawler = new Crawler();
         $crawler->add(1);
     }
```

Comment:
This test does not pass anymore because of the trigger added by the other PR. @stof Can you submit a PR to fix this? Thanks.